win32ss/user/ntuser/desktop.c

```diff
@@ -27,9 +27,26 @@ IntFreeDesktopHeap(IN PDESKTOP pdesk);
 
 /* GLOBALS *******************************************************************/
 
-/* These can be changed via CSRSS startup, these are defaults */
-DWORD gdwDesktopSectionSize = 512;
+/* These can be changed via registry settings.
+ * Default values (interactive desktop / non-interactive desktop):
+ * Windows 2003 x86: 3 MB / 512 KB (Disconnect: 64 KB, Winlogon: 128 KB)
+ * Windows 7 x86: 12 MB / 512 KB
+ * Windows 7 x64: 20 MB / 768 KB
+ * Windows 10 x64: 20 MB / 4 MB
+ * See:
+ * - https://dbmentors.blogspot.com/2011/09/desktop-heap-overview.html
+ * - https://www.ibm.com/support/pages/using-microsoft-desktop-heap-monitor-dheapmon-determine-desktop-heap-space-and-troubleshoot-filenet-p8-and-image-services-issues
+ * - https://www.betaarchive.com/wiki/index.php?title=Microsoft_KB_Archive/184802
+ * - https://kb.firedaemon.com/support/solutions/articles/4000086192-windows-service-quota-limits-and-desktop-heap-exhaustion
+ * - https://learn.microsoft.com/en-us/troubleshoot/windows-server/performance/desktop-heap-limitation-out-of-memory
+ */
```

Comment:
Can we still boot ReactOS to Desktop with just 49MB RAM assigned to the VM after this change? 0.4.14-release-122-g7d2cd22 can successfully boot to desktop **in bootcd and livecd** with just 49MB assigned.

![image](https://github.com/user-attachments/assets/effb5cf9-cb7e-4f45-8ed3-972f9ee44944)

(bootcd-mem-requirements are usually a bit higher than livecd's)

Reply:
<img width="394" alt="image" src="https://github.com/user-attachments/assets/b77a723c-b5c3-4cbe-a25d-5032b58df7f4">

Reply:
This change shouldn't affect memory requirements. Feel free to test.